styx-scheduler-service/src/main/java/com/spotify/styx/Scheduler.java

```diff
@@ -258,11 +258,11 @@ private void sendTimeout(WorkflowInstance workflowInstance) {
   }
 
   @AutoValue
-  abstract static class InstanceState {
-    abstract WorkflowInstance workflowInstance();
-    abstract RunState runState();
+  public abstract static class InstanceState {
```

Comment:
I'd prefer keeping this private.

In `KubernetesDockerRunner`, you can get the `WorkflowInstance` from the `RunState`, no need to spread this class across different component.